python/ql/test/experimental/library-tests/CallGraph/InlineCallGraphTest.ql

```diff
@@ -31,10 +31,8 @@ predicate typeTrackerCallEdge(CallNode call, Function callable) {
 predicate typeTrackerClassCall(CallNode call, Function callable) {
   exists(call.getLocation().getFile().getRelativePath()) and
   exists(callable.getLocation().getFile().getRelativePath()) and
-  exists(TT::NormalCall cc |
-    cc = TT::TNormalCall(call, _, any(TT::TCallType t | t instanceof TT::CallTypeClass)) and
-    TT::TFunction(callable) = TT::viableCallable(cc)
-  )
+  TT::resolveCall(call, callable, any(TT::TCallType t | t instanceof TT::CallTypeClass)) and
+  TT::TFunction(callable) instanceof TT::DataFlowFunction
```

Comment:
I'm not sure whether this line could be removed. I've kept it, since it constrains `callable`, but those constraints might be irrelevant - or even potentially undesirable?

Reply:
I think it would be fine to remove that line after you have rewritten the other parts of the code